.gitignore

```diff
@@ -9,3 +9,5 @@ venv
 .ipynb_checkpoints
 data/
 \.tmp/
+
+appyters/Enrichr_Consensus_Terms/.DS_Store
```

Comment:
Please instead just put a local .gitignore (in this directory, not modifying the global .gitignore) if you want to take this out. In general I'd recommend adding .DS_Store to `.git/info/exclude` instead but it's fine

Reply:
Ok so I did the latter lol